org.eclipse.xtext.xbase.lib/src/org/eclipse/xtext/xbase/lib/ArrayExtensions.java

```diff
@@ -195,6 +220,32 @@ public static boolean equals(boolean[] array, Object other) {
 		return array.clone();
 	}
 	
+	 /**
+     * Returns whether the array contains the given element.
+	 * @param index
+	 *            the index
+	 * @return the value at the given index
+     * 
+     * More formally, returns <tt>true</tt> if and only if this array contains
+     * at least one element <tt>e</tt> such that
+     * <tt>o==e</tt>.
+     *
+     * @param array
+     * 			the array to test
+     * @param o 
+     * 			element whose presence in this array is to be tested
+     * @return <tt>true</tt> if this array contains the specified element
+     */
+	@Pure
+	public static boolean contains(boolean[] array, boolean o) {
```

Comment:
`o` is not the typical name for a parameter of type boolean

Reply:
done
I renamed all the the scalar parameters in contains to 'value', also in the javadoc